developers/wallets/index.md

```diff
@@ -21,37 +21,21 @@ CHAPI integrates easily into digital wallet software, allowing your wallet to re
 
 * [Working with `QueryByExample` format requests](querybyexample)
 - **Example Code**: the [chapi-demo-wallet](https://github.com/credential-handler/chapi-demo-wallet) contains a full example implementation and is referenced throughout this guide.
-- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed functionality in the browser.
+- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed `CredentialHandler` API within the browser.
 - **Helper Library**: The [web-credential-handler](https://github.com/credential-handler/web-credential-handler) library provides helper functions for CHAPI integration in your code.
 
-## Wallet Registration
+## Web Wallet Registration
 
-Wallets need to be registered with the browser as a Credential Handler in order to store or retrieve Verifiable Credentials on the Web.
+All Wallets need to be registered as a Credential Handler in the user's browser in order to store or retrieve Verifiable Credentials (VCs) from across the Web.
 
-### 1. Import the CHAPI Polyfill into your wallet app
-If you're developing on Node.js, add the credential-handler-polyfill library to your project.  You can also install the web-credential-handler helper library to simplify your code.
+A Web Wallet is typically made up of three main parts:
+* `/index.html` - main page for the wallet
+* `/manifest.json` - Web App Manifest containting credential handler affordances
+* `/wallet-worker.html` - routing endpoint for passing URLs between CHAPI and the Wallet
 
-```sh
-npm i credential-handler-polyfill@3
-npm i web-credential-handler@2
-```
+### 1. Add a `credential_handler` to your Web app's `manifest.json`
 
-In your code, you can import and load the polyfill library as follows:
-
-```javascript
-import * as CredentialHandlerPolyfill from 'credential-handler-polyfill';
-import * as WebCredentialHandler from 'web-credential-handler';
-
-await CredentialHandlerPolyfill.loadOnce();
-console.log('Ready to work with credentials!');
-```
-<p class="code-annotation">
-  <a href="https://github.com/credential-handler/chapi-demo-wallet/blob/master/install-wallet.js"
-  target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/install-wallet.js </a>
-</p>
-
-### 2. Add a `credential_handler` to your app's manifest.json
-In order to register a credential handler, your web app must serve a "manifest.json" file from its root path ("/manifest.json"). This file must also be CORS-enabled.  At a minimum, add the following `credential_handler` object:
+In order to register a credential handler, your Web app must serve a `manifest.json` file from its root path (`/manifest.json`). This file _must_ be CORS-enabled. At a minimum, the `manifest.json` file should contain the following `credential_handler` object:
```

Comment:
We should add which CORS headers are required to help devs.

Reply:
@dlongley are their suggested ones? Likely don't want everyone taking the "easy" road and just `Access-Control-Allow-Origin: *` ...but we also don't want them having to chase the domain name around--should it change.

@brianorwhatever you've done this recently. Any suggestions here on CORS specifics?

Reply:
* is required as it could be any domain on the web that needs to pull this file down